src/frontend/ts/types/urbitChess.ts

```diff
@@ -42,7 +44,10 @@ export enum Action {
   OfferDraw = 'offer-draw',
   AcceptDraw = 'accept-draw',
   DeclineDraw = 'decline-draw',
-  Move = 'move'
+  Move = 'move',
+  ChangeSpecialDrawPreference = 'change-special-draw-preference',
+  ClaimSpecialDraw = 'claim-special-draw',
+  DeclineSpecialDraw = 'decline-special-draw'
```

Comment:
Don't need `DeclineSpecialDraw` - making a move instead of taking the draw is an implicit decline